locales/pt_BR/pt_BR.json

```diff
@@ -290,7 +290,7 @@
     "It looks like you do not have an active subscription. You may choose one of the subscription plans below to get started. Subscription plans may be changed or cancelled at your convenience.": "Parece que você não tem uma assinatura ativa. Você pode escolher um dos planos de assinatura abaixo para começar. Os planos de assinatura podem ser alterados ou cancelados conforme sua conveniência.",
     "Italy": "Itália",
     "Jamaica": "Jamaica",
-    "Jane Doe": "Jane Doe",
+    "Jane Doe": "Maria desconhecida",
```

Comment:
Is it correct that the surname is written with a lowercase letter?

Reply:
I'm sorry, I'm fixing it.